src/cosalib/gcp.py

```diff
@@ -1,13 +1,15 @@
 import os
+import re
 import urllib
 from cosalib.cmdlib import run_verbose
 from tenacity import (
     retry,
     stop_after_attempt
 )
 
+GCP_NAMING_RE = r"[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?|[1-9][0-9]{0,19}"
```

Comment:
Optional: link to GCP doc with regex would be neat

Reply:
That came from the API :)  But link added.